examples/resources/mondoo_integration_gitlab/resource.tf

```diff
@@ -0,0 +1,29 @@
+variable "gitlab_token" {
+  description = "The GitLab Token"
+  type        = string
+  sensitive   = true
+}
+
+provider "mondoo" {
+  space = "hungry-poet-123456"
+}
+
+# Setup the GitLab integration
+resource "mondoo_integration_gitlab" "gitlab_integration" {
+  name = "GitLab Integration"
+
+  # base_url = "" # optional, otherwise gitlab.com
+  # group = "" # optional, otherwise all groups are discovered
```

Comment:
I would remove these comments and instead, add the information in the schema so that it shows in the documentation.

Reply:
The comments for `# optional, otherwise gitlab.co`. and `# optional, otherwise all groups are discovered` are now in the schema